accelerator/message.h

```diff
@@ -68,6 +69,7 @@ static struct ta_cli_argument_s {
                           {"seed", SEED_CLI, "IOTA seed", OPTIONAL_ARG},
                           {"cache", CACHE, "Enable cache server with Y", REQUIRED_ARG},
                           {"config", CONF_CLI, "Read configuration file", REQUIRED_ARG},
+                          {"proxy_api", PROXY_API, "Set the way to pass proxy API", OPTIONAL_ARG},
```

Comment:
Maybe use passthrough or add directly before pass proxy API in the comment?
.

Reply:
but one of the way to pass proxy API doesn't transmit directly